dashboard/test/factories/pd_factories.rb

```diff
@@ -1144,7 +936,7 @@
       status :accepted
     end
 
-    association :pd_application, factory: :pd_facilitator1920_application
```

Comment:
What is the purpose of keeping this and changing it to `:pd_teacher_application` instead of just deleting it?

Reply:
Same for some of the other changes relating to FACILITATOR_APPLICATION_FACTORY --> TEACHER_APPLICATION_FACTORY

Reply:
For this particular association, this is for the `pd_fit_weekend1920_registration` factory (https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories/pd_factories.rb#L1142), which still exists because that model still exists (we "need" it for the data –– unclear the best approach to remove this model). Since we still want the model for now, it still needs the association with `pd_application`, and since we don't have the facilitator application factory any more, we have to use an application factory that exists, e.g. the teacher application factory.

For all the other changes, it's similar: we are testing functionality that still exists, so need to use a different way to test it. There's perhaps a cleaner solution with this, but I'm not sure what it is!